packages/components/src/Helpers.purs

```diff
@@ -6,20 +6,25 @@ import           Control.Alt ((<|>))
 import           Data.Date               (Date, adjust)
 import           Data.DateTime           (DateTime (..))
 import           Data.DateTime           as DateTime
+import           Data.Either             (hush)
 import           Data.Enum               (toEnum)
 import           Data.Formatter.DateTime (Formatter, FormatterCommand (..),
                                           format)
 import           Data.Int                (toNumber)
 import           Data.List               (fromFoldable)
-import           Data.Maybe              (Maybe (..), fromJust)
+import           Data.Maybe              (Maybe (..), fromJust, isJust)
 import           Data.String             (Pattern (..))
 import           Data.String             as String
+import           Data.String.Regex       as Regex
 import           Data.Time               (Time (..))
 import           Data.Time.Duration      (Days (..), Hours (..), negateDuration)
+import           Data.UUID               as UUID
 import           Effect                  (Effect)
 import           Effect.Now              as Now
 import           Partial.Unsafe          (unsafePartial)
 
+
```

Comment:
Extra space.

Reply:
All changes to this file undone, so fixed.